pkgs/development/python-modules/qpsolvers/default.nix

```
@@ -24,28 +30,38 @@ buildPythonPackage rec {
     hash = "sha256-/HLc9dFf9F/6W7ux2Fj2yJuV/xCVeGyO6MblddwIGdM=";
   };
 
-  pythonImportsCheck = [ "qpsolvers" ];
+  nativeBuildInputs = [
+    flit-core
+  ];
 
   propagatedBuildInputs = [
-    daqp
-    ecos
     numpy
-    osqp
     scipy
-    scs
   ];
 
   nativeCheckInputs = [
-    flit-core
-    quadprog
     unittestCheckHook
+  ] ++ passthru.optional-dependencies.solvers;
```

You can do
```suggestion
  ] ++ ++ lib.flatten (builtins.attrValues passthru.optional-dependencies);
```
if you need all solvers.